src/graphql/query/lendByCategory/lendByCategory.graphql

```diff
@@ -1,17 +1,6 @@
 query lendByCategory($basketId: String) {
+	hasEverLoggedIn @client
 	general {
-		mlServiceBandit: uiExperimentSetting(key: "EXP-ML-Service-Bandit-LendByCategory") {
-			key
-			value
-		}
-		mfiRecommendations: uiExperimentSetting(key: "mfi_recommendations") {
```

Comment:
didn't see this below, is it all clear to remove?

Reply:
thanks!

Reply:
@mcstover Are you referring to these `uiExperimentSetting` settings in general or specifically `mfi_recommendations`? Because we now prefetch all of the active settings, we no longer need to call `uiExperimentSetting` or assignment, unless a component needs the results as part of its own prefetch. And `mfi_recommendations` specifically is an experiment we just removed, but this was a left over piece.

Reply:
just making sure it was cruft and ok to be removed